src/components/Header/index.js

```diff
@@ -1,38 +1,53 @@
-/* (C) Copyright 2019 Hewlett Packard Enterprise Development LP. */
-import React from 'react';
-import { Box, Text, Header as HeaderGrommet, Image } from 'grommet';
+import React, { useContext, useState, useEffect } from 'react';
+import {
+  Box,
+  Button,
+  Text,
+  Header as HeaderGrommet,
+  Image,
+  ResponsiveContext,
+  Layer,
+  Form,
```

Comment:
remove unused import Form

Reply:
ill clean up the code in a separate branch